rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/RMServiceImpl.java

```diff
@@ -136,11 +128,7 @@ public void logout(String sessionId) throws ServiceException {
         }
     }
 
-    /*
-     * (non-Javadoc)
-     * 
-     * @see Service#login(java.lang.String, java.lang.String, java.io.File, java.lang.String)
-     */
+    @Override
     public String login(String login, String pass, File cred, String ssh) throws RestServerException, ServiceException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)